pkg/cmd/server/kubernetes/node/options/options.go

```diff
@@ -31,7 +31,7 @@ func ComputeKubeletFlags(startingArgs map[string][]string, options configapi.Nod
 	imageTemplate.Latest = options.ImageConfig.Latest
 
 	path := ""
-	var fileCheckInterval int64
+	fileCheckInterval := int64(20)
```

Comment:
do you want that or do you want to only conditionally set the flag at all?

Reply:
set conditionally is better. i'll update it.